src/tree_sitter/parser.h

```diff
@@ -25,16 +25,13 @@ typedef struct {
   bool named : 1;
 } TSSymbolMetadata;
 
-typedef struct TSLexer TSLexer;
-
-struct TSLexer {
+typedef struct {
+  void (*advance)(void *, bool);
+  void (*mark_end)(void *);
+  uint32_t (*get_column)(void *);
   int32_t lookahead;
   TSSymbol result_symbol;
-  void (*advance)(TSLexer *, bool);
-  void (*mark_end)(TSLexer *);
-  uint32_t (*get_column)(TSLexer *);
-  bool (*is_at_included_range_start)(TSLexer *);
-};
+} TSLexer;
```

Comment:
@maxbrunsfeld I was surprised to see this in the change set. Should this change be here, or is this header file change possibly related to some local development environment error?

Reply:
I think this is because you generated this using an older version of `tree-sitter-cli`.